CHANGELOG.md

```diff
@@ -1,3 +1,92 @@
+# 94.1.0
+
+This minor release intoduces new APIs to FlexibleHeader and Ripple, and updates the MDCSnackbar appearance to the latest style. The release also includes bug fixes for multiple components.
+
+## New Features
+
+### FlexibleHeader
+
+FlexibleHeader can now inform the receiver that its view's tracking scroll view has changed through the flexibleHeaderView:didChangeTrackingScrollViewAnimated: method. Example:
+
+**Swift**
+
+```swift
+class MyAnimationDelegate: NSObject, MDCFlexibleHeaderViewAnimationDelegate {
+	func flexibleHeaderView(_ flexibleHeaderView: MDCFlexibleHeaderView,
+                          didChangeTrackingScrollViewAnimated animated: Bool) {
+   if animated {
+      // success
+    }
+}
+```
+
+**Objective-C**
+
+```objc
+// MyAnimationDelegate conforms to: MDCFlexibleHeaderViewAnimationDelegate
+@interface MyAnimationDelegate
+   -(BOOL)flexibleHeaderView:(MDCFlexibleHeaderView *)flexibleHeaderView 
+   			didChangeTrackingScrollViewAnimated:(BOOL)animated {
+   	if (animated) {
+   		// success
+    }
+   }
+ @end
+```
+
+Additionally, you can now animate your own changes alongside the default animations of the flexible header, by passing an animation sequence and a completion block to animateWithAnimations:completion:. Example:
+
+**Swift**
+
+```swift
+myHeaderView.animate(animations: {
+    scrollView.contentOffset = CGPoint(x: 0, y: -100)
+    myHeaderView.maximumHeight = 100
+})
+```
+
+**Objective-C**
+
+```objc
+[myHeaderView animate:^{
+    scrollView.contentOffset = CGPointMake(0, -100.f);
+    myHeaderView.maximumHeight = 100.f;
+}];
+```
+
+### Ripple
```

Comment:
Please move this component change subsection into the "Component changes" section.

Reply:
Done